pkg/controllers/management/etcdbackup/etcdbackup.go

```diff
@@ -400,15 +425,44 @@ func (c *Controller) etcdRemoveSnapshotWithBackoff(b *v3.EtcdBackup) error {
 	})
 }
 
-// rotateExpiredBackups removes backups that are older than the expiration period, while retaining the desired number of etcd backups.
-// This function expects backups to be sorted from newest to oldest. In practice this function should only delete the last backup,
-func (c *Controller) rotateExpiredBackups(cluster *v3.Cluster, backups []*v3.EtcdBackup) error {
+func (c *Controller) rotateSuccessfulBackups(cluster *v3.Cluster) error {
```

Comment:
Nit: Add func descriptions to the funcs (at least the larger ones) that you've added. A comment was deleted here instead of modified

Reply:
Yes, I'll be flushing out the comments in a future PR (likely the one where I create the integration/validation tests)